server/routes/item.js

```diff
@@ -10,16 +10,16 @@ const {
 } = require('../controllers/item');
 const { authorize } = require('../middleware/authorize');
 
-// Add new product
+// Add new item
```

Comment:
get rid of all these comments - they do not introduce any additional information that can be already read from code